CMakeLists.txt

```diff
@@ -113,7 +111,7 @@ endif()
 # Thus, we give these flags to allow usage of the old interface in newer
 # versions of HDF5.
 if(NOT (${HDF5_VERSION} VERSION_LESS 1.12.0))
-  list(APPEND cxxflags -DH5Oget_info_by_idx_vers=1 -DH5O_info_t_vers=1)
+    add_compile_options(-DH5Oget_info_by_idx_vers=1 -DH5O_info_t_vers=1)
```

Comment:
The "modern" CMake paradigm is to do everything based on [targets](https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#targets-and-properties). See the recommendation [here](https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#forget-the-commands-add_compile_options-include_directories-link_directories-link_libraries) to not use add_compile_options.

Reply:
Ah interesting- I will remove this change then